app/controllers/follow_requests_controller.rb

```diff
@@ -19,7 +19,6 @@ def new
   def edit
```

Comment:
make sure you are setting the `:is_authorized_user` here at the top of the controller!